ui/pages/confirmations/hooks/useConfirmationAlerts.ts

```diff
@@ -37,25 +38,29 @@ function useTransactionAlerts(): Alert[] {
   const gasFeeLowAlerts = useGasFeeLowAlerts();
   const gasTooLowAlerts = useGasTooLowAlerts();
   const insufficientBalanceAlerts = useInsufficientBalanceAlerts();
+  const multipleApprovalAlerts = useMultipleApprovalsAlerts();
   const networkBusyAlerts = useNetworkBusyAlerts();
   const noGasPriceAlerts = useNoGasPriceAlerts();
+  const nonContractAddressAlerts = useNonContractAddressAlerts();
   const pendingTransactionAlerts = usePendingTransactionAlerts();
   const resimulationAlert = useResimulationAlert();
   const firstTimeInteractionAlert = useFirstTimeInteractionAlert();
```

Comment:
I see that in the original PR, this hook call was moved up to the 2nd place:  https://github.com/MetaMask/metamask-extension/pull/33237/files#diff-511cf66c68ef2df6eea211f7676e8d704301aa7ab1f9346e115a3d1f6601f741R35

Was that intentional? Is it important for it to be moved up? Maybe we should move it here as well to reduce conflicts when this PR is merged, unless there's a reason not to